core-backend/sandbox/src/env.rs

```diff
@@ -260,6 +260,8 @@ where
 
         builder.add_func(Alloc, wrap_common_func!(FuncsHandler::alloc, (1) -> (1)));
         builder.add_func(Free, wrap_common_func!(FuncsHandler::free, (1) -> (1)));
+        builder.add_func(IsGetterCalled, wrap_common_func!(FuncsHandler::forbidden, () -> ()));
```

Comment:
Why do we need this sys-calls support on back-end if they were removing on instrumentation stage ?

Reply:
I'm working on that. We probably need to add it to the list of forbidden functions. This causes the error right now: "Not all existing sys-calls were added to the module's env"